### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
### Context
Misspellings make it harder to search for things; they make it harder for people to understand code and descriptions; and sometimes they result in bugs.

### Contributor Checklist
- [x] [Review Contribution Guidelines](https://github.com/gradle/gradle/blob/master/CONTRIBUTING.md)
- [x] Make sure that all commits are [signed off](https://git-scm.com/docs/git-commit#git-commit---signoff) to indicate that you agree to the terms of [Developer Certificate of Origin](https://developercertificate.org/).
- [x] Check ["Allow edit from maintainers" option](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) in pull request so that additional changes can be pushed by Gradle team
- [ ] Provide integration tests (under `<subproject>/src/integTest`) to verify changes from a user perspective
- [ ] Provide unit tests (under `<subproject>/src/test`) to verify logic
- [ ] Update User Guide, DSL Reference, and Javadoc for public-facing changes
- [ ] Ensure that tests pass locally: `./gradlew <changed-subproject>:check`

### Gradle Core Team Checklist
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation
- [ ] Recognize contributor in release notes

I'm happy to drop files/directories if they're not yours -- I can't figure out if `subprojects/` is third party content.
I'm happy to split into, e.g. API, docs, code as requested.

--
Generated by https://github.com/jsoref/spelling `f` -- this is the misspelled `a` family

In short, my tool identifies candidates, I manually select and inspect candidates, and corrections are chosen by me manually (although I often let Google suggest the correction instead of typing it myself as it has a lower typo rate).

### CODE REVIEW

**Comment by blindpirate** [comment]
@jsoref Thank you very much for the contribution! I'm quite impressed by your carefulness. Some suggestions:

- Please fix the failed DCO.
- Please don't put so many changes in one PR. Nobody can review 1000+ files changes. On my computer, this page even scrolls slowly because of the commit number. I'd suggest at most 10~20 files per PR.
- Due to historical reason, we'd prefer not to change `executer` to `executor`. Not to mention some `-Dorg.gradle.integtest.executor` in system property, which is binding to our CI configurations. My suggestion would be only changing the incorrect spells in comments, local variables, etc.. Please don't touch the code which affects the program functionality.

We'd be happy to review and merge this when you split it into smaller pieces. Out of curiosity, did you do the spelling check manually or by some automatic tools?

**Comment by jsoref** [comment]
I omitted my standard header, but here it is:
Generated by https://github.com/jsoref/spelling `f`; to maintain your repo, please consider `fchurn`

In short, my tool identifies candidates, I manually select and inspect candidates, and corrections are chosen by me manually (although I often let Google suggest the correction instead of typing it myself as it has a lower typo rate).

I'll look into splitting this, but probably not before Sunday.

Splitting on file count is doable, and certainly if I'm skipping things that impact functionality splitting that way entails less risk to splits because things are less dependent. But do note that it's generally easier for me to have all instances of a word correction together as people generally object to a correction as a whole (e.g. executor).

I'll fix the DCO of course.

As an aside, 180 is below the 250 commit cut off that GitHub has and I'm slowly merging what was effectively 350 commits split into pieces.

**Comment by blindpirate** [comment]
Thanks! I'm not saying splitting on file count, I'm just suggesting splitting to a reasonable PR size - for example, if you prefer correcting word by word (i.e. each wrong word per PR) and probably a wrong word appears in 100 files - that's acceptable, too. 

I'd love to help you improve the spelling and make sure nothing broken.

**Comment by jsoref** [comment]
(Each wrong word per PR would be 180 PRs, which is a bit much let's see what 10-20 words looks like.)

**Comment by jsoref** [comment]
`DCO Successful in -1m — DCO` that seems buggy.

**Comment by jsoref** [comment]
@blindpirate: how's this?

**Comment by blindpirate** [comment]
Great! Thank you very much!

I expect there would be no harm to fix these typos. Anyway, once CI checks complete.

**Comment by blindpirate** [comment]
Thanks @jsoref for the contribution!
